initrd/bin/kexec-select-boot

```diff
@@ -48,6 +48,22 @@ bootdir="${bootdir%%/}"
 paramsdev="${paramsdev%%/}"
 paramsdir="${paramsdir%%/}"
 
+PRIMHASH_FILE="$paramsdir/kexec_primhdl_hash.txt"
+if [ "$CONFIG_TPM2_TOOLS" = "y" ]; then
+	if [ -r "$PRIMHASH_FILE" ]; then
+		sha256sum -c "$PRIMHASH_FILE" \
+			|| {
+			echo "FATAL: Hash of TPM2 primary key handle mismatch!";
```

Comment:
This is thrown all the time until a TPM disk unlock key is set, which doesn't unseal properly as of now